### PR TITLE
Fixes GermanVoltageLevel.parse bug

### PR DESCRIPTION
**Pull request by denstre**
Adresses #118 
Please read the comment to decide whether you can work on another solution

### CODE REVIEW

**Comment by ckittl** [comment]
@johanneshiry 
Please consider my thoughts in [my issue comment](https://github.com/ie3-institute/PowerSystemDataModel/issues/118#issuecomment-624462643). As you raised the "bug", I would like to have your thoughts on this.

File: src/test/groovy/edu/ie3/datamodel/models/CommonVoltageLevelTest.groovy [hunk 1]
```diff
@@ -48,13 +48,4 @@ class CommonVoltageLevelTest extends Specification {
 		expect:
 		!dut.covers("HS", Quantities.getQuantity(110d, KILOVOLT))   // May be false
 	}
-
```
Comment (ckittl) [inline src/test/groovy/edu/ie3/datamodel/models/CommonVoltageLevelTest.groovy:51]: Please alter this test, as not the function has gone, but become a different output. Please test for the return of `false`.

Comment (ckittl) [reply src/test/groovy/edu/ie3/datamodel/models/CommonVoltageLevelTest.groovy:51]: The way you resolve this, obviously depends on the outcome of the discussion of the issue.

Comment (johanneshiry) [reply src/test/groovy/edu/ie3/datamodel/models/CommonVoltageLevelTest.groovy]: Is this still valid @ckittl? I think this can be resolved can't it?

**Pull request merged**